src/components/controls/radio-buttons/RadioGroup.js

```diff
@@ -13,10 +15,11 @@ function RadioGroup({
   hasError = false,
   disableAllOptions = false,
   inline = true,
-  onClick = noop
+  onClick = noop,
+  theme
 }) {
   return (
-    <Fieldset legendText={legendText}>
+    <Fieldset legendText={legendText} theme={theme}>
```

Comment:
If we wrap this Fieldset in a ThemeProvider and pass the theme prop to it instead of here, it will pass it down automatically and better matches the ThemeProvider pattern we're using elsewhere.